### PR TITLE
[FIX] pos_restaurant: prevent traceback on added table

### PR DESCRIPTION
**Pull request by lse-odoo**
To reproduce (v16):
 1. Install PoS Restaurant
 2. Open the restaurant in a device D1
 3. Open the restaurant in a device D2
 4. From D1 create a new table 

=> Traceback will keep appearing on D2:
`cannot read propoertied of undefined (reading 'id')`

This is a consequence of trying to set the order count on a table which is not synced (and will never be in 16 until a refresh)

After this commit:
 A console warning is logged and the POS proceed ignoring this missing tables

opw-3850899

### CODE REVIEW

**Comment by lse-odoo** [comment]
Note: it was not reproducible in 17 (because of websocket probably).

Checking the highest version with the issue...

File: addons/pos_restaurant/static/src/js/Screens/FloorScreen/FloorScreen.js [hunk 1]
```diff
@@ -162,6 +179,7 @@ odoo.define('pos_restaurant.FloorScreen', function (require) {
                     throw error;
                 }
             }
+
```
Comment (qle-odoo) [inline addons/pos_restaurant/static/src/js/Screens/FloorScreen/FloorScreen.js:182]: useless diff

**Comment by caburj** [comment]
fw-bot ignore

**Comment by caburj** [comment]
robodoo delegate+

**Comment by lse-odoo** [comment]
@robodoo r+

**Pull request closed without merging**